### ISSUE TITLE
[PAGE ISSUE]: 'Migrating to null safety'

### ISSUE THREAD

**Issue by iLoveDocs** [dart-lang/site-www#4363]
### Page URL

https://dart.dev/null-safety/migration-guide.html

### Page source

https://github.com/dart-lang/site-www/tree/main/src/null-safety/migration-guide.md

### Describe the problem

In the [Migrate](https://dart.dev/null-safety/migration-guide#step2-migrate) section, there's a line: 

> A named parameter that shouldn’t be nullable needs to be [marked required](https://dart.dev/null-safety/understanding-null-safety#required-named-parameters).

As there was no mention of "named parameter" in the given context, this line doesn't make much sense. 

### Expected fix

If we need to mention the use of non-nullable type in the named parameter, we should also add a line similar to below to make the above context complete. 

> A named parameter that shouldn't be nullable either needs to be marked required or provided a default value. 

In other words, we should either remove the used line in docs or add the above line (in different words) to complete it.
